app/logic/student_invitation_texts.rb

```diff
@@ -0,0 +1,143 @@
+# frozen_string_literal: true
+
+class StudentInvitationTexts < InvitationTexts
+  class << self
+    def nth_response_pool(curidx)
+      sms_pool = []
+      if curidx.zero? # Voormeting
+        sms_pool += [
+          'Welkom bij de kick-off van het onderzoek ‘u-can-act’. Fijn dat je meedoet! ' \
+          'Vandaag starten we met een aantal korte vragen, morgen begint de wekelijkse vragenlijst. ' \
+          'Via de link kom je bij de vragen en een filmpje met meer info over u-can-act. Succes!'
+        ]
+      end
+      if curidx == 1 # Eerste wekelijkse vragenlijst
+        sms_pool += [
+          'Vul jouw eerste wekelijkse vragenlijst in en verdien twee euro!'
+        ]
+      end
+      sms_pool
+    end
+
+    # rubocop:disable Metrics/MethodLength
+    # rubocop:disable Metrics/CyclomaticComplexity
+    def rewards_threshold_pool(threshold)
+      case threshold
+      when 1000 # 10 euro
+        [
+          'Whoop! Na deze vragenlijst heb je €10,- verdiend. Ga zo door!'
+        ]
+      when 2000 # 20 euro
+        [
+          'Je gaat hard {{deze_student}}! Na deze vragenlijst heb je al €20,- gespaard.'
+        ]
+      when 3000 # 30 euro
+        [
+          'De teller blijft lopen! Na deze vragenlijst passeer jij de €30 ☺'
+        ]
+      when 4000 # 40 euro
+        [
+          'Hé {{deze_student}}. Door jouw goede inzet heb je bijna €40,- verdiend!'
+        ]
+      when 5000 # 50 euro
+        [
+          'Geweldig, na deze vragenlijst heb je al €50,- verdiend!'
+        ]
+      when 6000 # 60 euro
+        [
+          'Door jouw fantastische hulp heb jij al bijna €60,- verdiend!'
+        ]
+      when 7000 # 70 euro
+        [
+          'Weet jij al wat je gaat doen met de €70,- die jij na deze vragenlijst hebt verdiend?'
+        ]
+      when 8000 # 80 euro
+        [
+          'Wat heb jij je al ontzettend goed ingezet {{deze_student}}! Inmiddels heb je bijna €80,- verdiend.'
+        ]
+      else
+        []
+      end
+    end
+
+    # rubocop:enable Metrics/CyclomaticComplexity
+    # rubocop:enable Metrics/MethodLength
+
+    def default_pool
+      # TODO: make sure to replace newlines by spaces in the email template.
```

Comment:
willen we nog checken bij @nrsnell of die newlines niet gewoon weg mogen? Dan is de tekst in de mail precies het zelfde als de tekst in de emails

Reply:
(en als het heel bewust is gedaan dan moeten we hier even een ticket voor maken / het ergens opschrijven dat we dit moeten doen)

Reply:
Ik heb er een ticket van gemaakt.